charts/cray-certmanager/templates/cert-manager/securityContext-job.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ template "cray-certmanager.fullname" . }}-cert-manager-{{ .Release.Revision }}"
+  labels:
+    app.kubernetes.io/name: {{ template "cray-certmanager.fullname" . }}-cert-manager
```

Comment:
This job should be set up as a post-install and post-upgrade hook. See the Helm documentation.

Reply:
https://helm.sh/docs/topics/charts_hooks/